diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/IndentationRule.kt

```diff
@@ -195,6 +187,19 @@ class IndentationRule(private val configRules: List<RulesConfig>) : Rule("indent
         }
     }
 
+    private fun ASTNode.getExceptionalIndentInitiator() = treeParent.let { parent ->
+        when (parent.psi) {
+            // fixme: custom logic for determining exceptional indent initiator, should be moved elsewhere
+            is KtDotQualifiedExpression -> {
```

Comment:
braces not needed?
and yes I would also move it away

Reply:
At first there was hack for only one case, but now that there are several I think it should be extracted to `CheckResult`, for example. I think I'll better do it in the next PR, because this is for sure not the last bug in indentation logic.